package.json

```diff
@@ -14,6 +14,7 @@
     "chart.js": "^2.9.3",
     "clone-deep": "^4.0.1",
     "debounce": "^1.2.0",
+    "moment": "^2.27.0",
```

Comment:
Out of curiosity, the moment package has a good replacement (date-fns), which is smaller, functional (easier to be tree-shaken) but still reliable =]
https://bundlephobia.com/result?p=moment
https://bundlephobia.com/result?p=date-fns

Reply:
moment.js is already being used by one of the dependencies.

Reply:
Yeah, moment.js is a dependency of chart.js so we're not actually increasing the bundle size by adding it. That said, at some point we will need to split our bundle to offload some of the large packages that are only used in a few places (like chart.js). Our bundle size is getting very big already. In that case it will be useful to explore alternatives.